src/com/mendix/widget/carousel/Carousel.ts

```diff
@@ -15,7 +15,8 @@ class CarouselDojo extends WidgetBase {
     staticImages?: StaticImage[];
 
     update(object: mendix.lib.MxObject, callback: Function) {
-        render(createElement(Carousel, { images: this.staticImages }), this.domNode);
+        const images = this.staticImages.map(image => ({ url : image.imageUrl }));
```

Comment:
And still `imageUrl`, although you might've changed that in another commit that I haven't seen yet :).